src/model/operation/transform.js

```diff
@@ -2304,9 +2304,7 @@ function _makeMoveOperationsFromRanges( ranges, targetPosition ) {
 		const op = new MoveOperation(
 			range.start,
 			range.end.offset - range.start.offset,
-			// If the target is the end of the move range this operation doesn't really move anything.
-			// In this case, it is better for OT to use range start instead of range end.
-			targetPosition.isEqual( range.end ) ? range.start : targetPosition,
+			targetPosition,
```

Comment:
I assume that maybe at some point it was important to change the target position, but then we might have added some cases and this got fixed in a different place. It didn't cause any test to fail and in this scenario, it looked like an obvious error (to use `range.start` instead of `targetPosition`).